### PR TITLE
updated async readme

### PR DESCRIPTION
**Pull request by RichardWright**
Feedback appreciated on this..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ bcrypt.compareSync(someOtherPlaintextPassword, hash); // false
 ```
 
 ### Why is async mode recommended over sync mode?
-If you are using bcrypt on a simple script, using the sync mode is perfectly fine. However, if you are using bcrypt on a server, the async mode is recommended. This is because the hashing done by bcrypt is CPU intensive, so the sync version will block the event loop and prevent your application from servicing any other inbound requests or events.
+If you are using bcrypt on a simple script, using the sync mode is perfectly fine. However, if you are using bcrypt on a server, the async mode is recommended. This is because the hashing done by bcrypt is CPU intensive, so the sync version will block the event loop and prevent your application from servicing any other inbound requests or events. The async version also uses another thread so as not to block the event loop.
```
Comment (defunctzombie) [inline README.md]: `The async version uses a thread pool which does not block the main event loop.`

**Comment by agathver** [comment]
@RichardWright, 
Updates?

**Comment by RichardWright** [comment]
@agathver I'll update my pr now.

**Comment by RichardWright** [comment]
@agathver done

**Comment by ncb000gt** [comment]
@RichardWright can you resolve the conflicts here and we can merge this in?

**Comment by RichardWright** [comment]
@ncb000gt fixed

**Comment by ncb000gt** [comment]
Thanks!

**Pull request merged**